graylog2-server/src/main/java/org/graylog2/rest/resources/search/AbsoluteSearchResource.java

```diff
@@ -62,8 +65,10 @@
     private static final Logger LOG = LoggerFactory.getLogger(AbsoluteSearchResource.class);
 
     @Inject
-    public AbsoluteSearchResource(Searches searches, ClusterConfigService clusterConfigService) {
-        super(searches, clusterConfigService);
+    public AbsoluteSearchResource(Searches searches,
+                                  ClusterConfigService clusterConfigService,
+                                  DecoratorProcessor decoratorProcessor) {
```

Comment:
Once merged this will require changes in https://github.com/Graylog2/graylog-plugin-map-widget/blob/03b07372fc5f8f097bf0327adfc92df2920ee39b/src/main/java/org/graylog/plugins/map/rest/MapDataResource.java#L56-L56 as well